agent/php_user_instrument_util_hashmap.h

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright 2022 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#if LOOKUP_METHOD == LOOKUP_USE_UTIL_HASHMAP
```

Comment:
This module implements an experimental lookup method and will probably be removed completely in the final version of this PR. It's left in for now as a reference.